src/PasswordField.js

```diff
@@ -102,9 +102,11 @@ class PasswordField extends React.Component {
       errorText,
       errorStyle,
       textFieldStyle,
-      style,
+      style: {
+        width,
```

Comment:
I just realized that this line broke `fullWidth`. It's fixed in the latest release.